frontend_tests/node_tests/user_events.js

```diff
@@ -75,14 +75,14 @@ run_test('updates', () => {
     };
     people.add_active_user(isaac);
 
-    user_events.update_person({user_id: isaac.user_id, is_guest: true});
+    user_events.update_person({user_id: isaac.user_id, role: 600});
```

Comment:
This are better written as e.g. `settings_config.user_role_values.guest.code` for readability.